src/sentry/static/sentry/app/views/organizationGroupDetails/groupMerged/mergedList.tsx

```diff
@@ -1,37 +1,53 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import PropTypes from 'prop-types';
 
 import EmptyStateWarning from 'app/components/emptyStateWarning';
 import Pagination from 'app/components/pagination';
 import {Panel} from 'app/components/panels';
 import QueryCount from 'app/components/queryCount';
 import {t} from 'app/locale';
-import SentryTypes from 'app/sentryTypes';
+import {Fingerprint} from 'app/stores/groupingStore';
+import {Project} from 'app/types';
 
 import MergedItem from './mergedItem';
 import MergedToolbar from './mergedToolbar';
 
-class MergedList extends React.Component {
-  static propTypes = {
```

Comment:
We shouldn't remove this yet because this is used by `groupMerged/index.jsx` (which is not in typescript).

Reply:
I've added back the `static propTypes` as you asked for.